quinn-proto/src/coding.rs

```diff
@@ -1,18 +1,25 @@
+//! Coding related traits.
+
 use std::net::{Ipv4Addr, Ipv6Addr};
 
 use bytes::{Buf, BufMut};
 use thiserror::Error;
 
 use crate::VarInt;
 
+/// Unexpected end of buffer error.
 #[derive(Error, Debug, Copy, Clone, Eq, PartialEq)]
 #[error("unexpected end of buffer")]
 pub struct UnexpectedEnd;
 
+#[doc(hidden)]
```

Comment:
Why is this hidden? It's part of `Codec`'s API.

Reply:
oversight, unhidden